basics/data-import/batch-ingestion/hadoop.md

```diff
@@ -40,7 +40,7 @@ Finally execute the hadoop job using the command -
 
 ```
 export PINOT_VERSION=0.8.0
```

Comment:
Let's update the version to `0.10.0` since this new build dir only applies to the latest version